src/sentry/receivers/project_detectors.py

```diff
@@ -44,6 +54,43 @@ def create_project_detectors(instance, created, **kwargs):
             sentry_sdk.capture_exception(e)
 
 
+def create_metric_detector_with_owner(project: Project, user=None, user_id=None, **kwargs):
+    """
+    Creates default metric detector when project is created, with the team as owner.
+    This listens to project_created signal which provides user information.
+    """
+
+    if not features.has("organizations:default-anomaly-detector", project.organization, actor=user):
+        return
+
+    owner_team = project.teams.first()
```

Comment:
maybe i haven't made a new project in sentry in a while but do we assign projects to team at creation? i can't remember doing that in the past when i created new projects